components/printful/actions/calculate-shipping-rates/calculate-shipping-rates.mjs

```diff
@@ -0,0 +1,90 @@
+import { parseObject } from "../../common/utils.mjs";
+import printful from "../../printful.app.mjs";
+
+export default {
+  key: "printful-calculate-shipping-rates",
+  name: "Calculate Shipping Rates",
+  description: "Fetches available shipping rates for a given destination. [See the documentation](https://developers.printful.com/docs/#tag/Shipping-Rate-API/operation/calculateShippingRates)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    printful,
+    storeId: {
+      propDefinition: [
+        printful,
+        "storeId",
+      ],
+    },
+    address1: {
+      label: "Address 1",
+      description: "The address 1",
+      type: "string",
+    },
+    city: {
+      label: "City",
+      description: "The city",
+      type: "string",
+    },
+    stateCode: {
+      label: "State Code",
+      description: "The state code. E.g. `SC`",
+      type: "string",
+    },
+    countryCode: {
+      label: "Country Code",
+      description: "The country code. E.g. `BR`",
+      type: "string",
+    },
+    zip: {
+      label: "ZIP/Postal Code",
+      description: "The ZIP/postal code. E.g. `89221525`",
+      type: "string",
+    },
+    phone: {
+      label: "Phone",
+      description: "The phone number",
+      type: "string",
+      optional: true,
+    },
+    items: {
+      type: "string[]",
+      label: "Items",
+      description: "A list of items in JSON format. **Example: [{\"variant_id\": \"123456\", \"external_variant_id\": \"123456\", \"quantity\": 123, \"value\": \"12345\" }]**",
+    },
+    currency: {
+      type: "string",
+      label: "Currency",
+      description: "3 letter currency code (optional), required if the rates need to be converted to another currency instead of store default currency",
+      optional: true,
+    },
+    locale: {
+      type: "string",
+      label: "Locale",
+      description: "Locale in which shipping rate names will be returned. Available options: `en_US` (default), `es_ES`",
```

Comment:
These should be `options` in the prop instead of being listed in the description